### PR TITLE
Se agrega el componente trabajadores, con su respectivo enrutamiento.

### PR DESCRIPTION
**Pull request by MarcosLoyo**
Agrego a Eric y a Julio para revisión

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -6,6 +6,10 @@ const routes: Routes = [
     path: 'home',
     loadChildren: './views/layout/layout.module#LayoutModule'
   },
+  {
+    path: 'workers',
```
Comment (JulioPerezAG) [inline src/app/app-routing.module.ts:10]: La ruta se debe agregar en el módulo de Layout en ves de aquí

**Review by JulioPerezAG — CHANGES_REQUESTED** [review]

File: src/app/app-routing.module.ts (hunk 1 above)
Comment (ericAlfonso) [inline src/app/app-routing.module.ts:10]: if you are using this module in layout module you dont have to add it here.. delete

File: src/app/views/layout/layout.module.ts [hunk 2]
```diff
@@ -3,6 +3,7 @@ import { CommonModule } from '@angular/common';
 import { LayoutComponent } from './layout.component';
 import { NavbarModule } from '../../components/navbar/navbar.module';
 import { RouterModule } from '@angular/router';
+import { WorkersComponent } from '../workers/workers.component';
```
Comment (ericAlfonso) [inline src/app/views/layout/layout.module.ts]: You are not using this import .. delete it

File: src/app/views/layout/layout.module.ts [hunk 3]
```diff
@@ -3,6 +3,7 @@ import { CommonModule } from '@angular/common';
...
 
 const routes = [{ path: '', component: LayoutComponent, children: [] }];
```
Comment (ericAlfonso) [inline src/app/views/layout/layout.module.ts]: You have to add your module worker module here

File: src/app/views/workers/workers.component.html [hunk 4]
```diff
@@ -0,0 +1,74 @@
+<app-navbar></app-navbar>
+<p>
```
Comment (ericAlfonso) [inline src/app/views/workers/workers.component.html]: delete this paragraph

File: src/app/views/workers/workers.component.html [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+  workers works!
+</p>
+<div class="container" style="padding:20px; background: rgb(33, 150, 243);">
```
Comment (ericAlfonso) [inline src/app/views/workers/workers.component.html]: the styles have to be in the workers.component.css

File: src/app/views/workers/workers.component.html [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+  <div class="row" style="padding:50px; background: white; margin: 10px 20px; ">
```
Comment (ericAlfonso) [inline src/app/views/workers/workers.component.html]: ditto

**Review by ericAlfonso — CHANGES_REQUESTED** [review]
good job

**Pull request closed without merging**